tests/e2e-migration/e2e_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/gomega"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/storage/drivers"
+	"k8s.io/kubernetes/test/e2e/storage/testsuites"
+
+	// ensure that cloud provider is loaded
+	_ "k8s.io/kubernetes/test/e2e/framework/providers/aws"
+)
+
+const kubeconfigEnvVar = "KUBECONFIG"
+
+func init() {
+	// k8s.io/kubernetes/test/e2e/framework requires env KUBECONFIG to be set
+	// it does not fall back to defaults
+	if os.Getenv(kubeconfigEnvVar) == "" {
+		kubeconfig := filepath.Join(os.Getenv("HOME"), ".kube", "config")
+		os.Setenv(kubeconfigEnvVar, kubeconfig)
+	}
+	framework.RegisterCommonFlags(flag.CommandLine)
+	framework.RegisterClusterFlags(flag.CommandLine)
+	_ = flag.Set("storage.migratedPlugins", "kubernetes.io/aws-ebs")
+	_ = flag.Set("provider", "aws")
+	flag.Parse()
```

Comment:
TODO for me. report-dir is getting lost somewhere here. It does work for the other e2e tests (single-az)

Reply:
it's working, just all tests are being skipped ~_~
```
Running Suite: AWS EBS CSI Driver End-to-End Tests
==================================================
Random Seed: 1566525635 - Will randomize all specs
Will run 37 specs
Running in parallel across 32 nodes
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
Ran 0 of 37 Specs in 0.579 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 37 Skipped 
```